packages/ui/react-ui-grid/src/Grid/Grid.tsx

```diff
@@ -109,7 +112,9 @@ const GridContent = forwardRef<NaturalDxGrid, GridScopedProps<GridContentProps>>
   const { id, editing, setEditBox, setEditing } = useGridContext(GRID_CONTENT_NAME, props.__gridScope);
   const [dxGrid, setDxGridInternal] = useState<NaturalDxGrid | null>(null);
 
-  // NOTE(thure): using `useState` instead of `useRef` works with refs provided by `@lit/react` and gives us a reliable dependency for `useEffect` whereas `useLayoutEffect` does not guarantee the element will be defined.
+  // TODO(burdon): Can we use useImperativeHandle here?
```

Comment:
I believe I demonstrated that `useImperativeHandle` causes a regression to the original issue; consumers still need to use `useState` to get a reliable effect dependency.

Reply:
let's discuss today.

Reply:
Even if this method can be made to be equivalent, what value does it deliver exactly?